pkgs/applications/editors/deadpixi-sam/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, freetype, libX11, libXt, libXft
+}:
+
+stdenv.mkDerivation rec {
+  name = "deadpixi-sam-unstable";
+  version = "2016-09-05";
+    src = fetchFromGitHub {
+      owner = "deadpixi";
+      repo = "sam";
+      rev = "6983187e2589647c087aab2825b99b014bae5dc3";
+      sha256 = "1agrs6k3kdpw672x0n5y2fdbrmkfxcfv1bi30lh5q75g0a4r42n9";
+    };
+
+  postPatch = ''
+    substituteInPlace config.mk.def \
+      --replace "/usr/include/freetype2" "${freetype.dev}/include/freetype2"
+  '';
+
+  makeFlags = [ "DESTDIR=$(out)" ];
+  buildInputs = [ freetype.dev libX11 libXt libXft ];
```

Comment:
I get the impression that `freetype.dev` is added here only because it is referenced in `postPatch`. If that is the case, you can safely remove it from build inputs.